snapcraft/elf/_patcher.py

```diff
@@ -67,6 +67,9 @@ def patch(self, *, elf_file: ElfFile) -> None:
             current_rpath = self.get_current_rpath(elf_file)
             proposed_rpath = self.get_proposed_rpath(elf_file)
 
+            emit.debug(f"Patcher.patch: current_rpath={current_rpath!r}")
+            emit.debug(f"Patcher.patch: proposed_rpath={proposed_rpath!r}")
```

Comment:
this should be on line and probably also log the file being operated on

Reply:
Displaying rpath and interpreter changes in verbose mode (changed `debug` to `progress`), and changed the output layout a little bit for readability. It will output as:
```
Patch ELF file: 'usr/bin/toilet'
  Interpreter='/snap/core22/current/lib64/ld-linux-x86-64.so.2'
  Current rpath=[]
  Proposed rpath=['$ORIGIN/../lib/x86_64-linux-gnu', '/snap/core22/current/lib/x86_64-linux-gnu']       
Patch ELF file: 'usr/lib/x86_64-linux-gnu/caca/libgl_plugin.so.0.0.0'
  Current rpath=[]
  Proposed rpath=['$ORIGIN/..', '/snap/core22/current/lib/x86_64-linux-gnu']
Patch ELF file: 'usr/lib/x86_64-linux-gnu/caca/libx11_plugin.so.0.0.0'
  Current rpath=[]
  Proposed rpath=['$ORIGIN/..', '/snap/core22/current/lib/x86_64-linux-gnu']
```